CHANGELOG.md

```diff
@@ -68,6 +68,19 @@ Interchain Security v3 uses SDK 0.47 and IBC 7.
 * `[x/ccv/provider]` (fix) [#977](https://github.com/cosmos/interchain-security/pull/977) Avoids panicking the provider when an unbonding delegation was removed through a `CancelUnbondingDelegation` message.
 * `[x/ccv/democracy]` (feat) [#1019](https://github.com/cosmos/interchain-security/pull/1019) Whitelisting non-legacy params in the "democracy module" require the entire module to be whitelisted. 
 
+## v2.1.0-provider-lsm
+
+Date: September 15th, 2023
```

Comment:
Note this date isn't present in the [v2.1.0-lsm changelog](https://github.com/cosmos/interchain-security/blob/v2.1.0-provider-lsm/CHANGELOG.md).